app/views/assignments/index.html.erb

```diff
@@ -1,15 +1,25 @@
 <h1><%= @current_office.name %> Kitchen Schedule</h1>
 
-<table>
+<br>
```

Comment:
`h1` and `table` are already block level elements meaning that this spacing is style rather than structure. Seems okay for this project but I would generally recommend adding styles to one or both of the surrounding elements instead.